src/execute-js-func.js

```diff
@@ -346,6 +350,51 @@ export function createProxyForRemote(windowOrWebView) {
 }
 
 /**
+ * Creates an object that is a representation of a module in the main process,
+ * but with all of its methods Promisified.
+ *
+ * @param {String} moduleName The name of the main process module to proxy
+ * @returns {Object}          A Proxy object that will invoke methods remotely.
+ *                            All methods will return a Promise.
+ */
+export function createProxyForMainProcessModule(moduleName) {
```

Comment:
Hm, instead of reimplementing all of this for MainProcessModule, it might be better to just have a Magic Global that the receiver knows about, i.e. createProxyForMainProcessModule is just really:

```js
function createProxyForMainProcessModule(moduleName) {
  return createProxyForRemote(null).__module__.[moduleName];
}
```

Then on the receiving side, you recognize `methodChain[0]`'s Magic Words, you require `methodChain[1]`, then do the same code. This implementation would stop us from having two completely separate code paths, especially in security-sensitive code like this